src/services/data/schemas.ts

```diff
@@ -24,7 +24,10 @@ export default (
 ): DataSchemasService => {
   async function find(options) {
     const result: PagedResult<Schema> = (
-      await client.get(httpAuth, `/${options?.rql || ''}`, options)
+      await client.get(httpAuth, `/${options?.rql || ''}`, {
+        ...options,
+        customResponseKeys: ['data.properties', 'data.statuses'],
```

Comment:
The argument Friday was to limit the changes needed everywhere.

Little improvements like the one mentioned above by Chad or for instance moving the adding of rql a layer down (WHY IS `${options?.rql || ''` EVERYWHERE?! sorry), could bring the complexity and meat of these endpoints and their option objects down quite a lot. These custom fields have more right to be here than rql and stuff like `addTransitionHelpersToSchema`, imho.

Lots of text but I do not have a strong opinion here though! Either way is fine.

Long way of saying: I agree, but if it is "extra work" I think we would be solving the wrong part of the problem?

Reply:
Agreed ^^ it's just once we commit to this we have 5+ more tickets to do, I just wanted to make sure we're aligned on how we want to do it rather than regretting it in a few days.

I agree with the rql options and MANY other things.